dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -64,8 +64,8 @@ def enroll_user_in_unit_with_learning_modules(learning_modules)
     end
   end
 
-  def focus_area_positions
-    plc_module_assignments.map {|a| a.plc_learning_module.stage.absolute_position unless a.plc_learning_module.required?}.compact
```

Comment:
I think flex categories could still mess us up, but I could be missing something.

Either way, I don't think that using position as an identifier when we already have a perfectly good unique identifier right there makes sense.